### PR TITLE
Add Ember Testing Style Guides

### PR DESCRIPTION
**Pull request by seanpdoyle**

### CODE REVIEW

**Comment by drapergeek** [comment]
I agree with everything here.

**Comment by BlakeWilliams** [comment]
:+1:

**Comment by seanpdoyle** [comment]
@drapergeek @BlakeWilliams @gabebw how do we feel about the wording here?

**Comment by seanpdoyle** [comment]
Also, I know that there is a majority preference towards mocha style `describe` blocks, but I felt the framework code was unimportant, and the sample code to focus on is in the body of the block.

Feel free to let me know if you think that's out of line

**Comment by BlakeWilliams** [comment]
I think it's fine in this case since it's framework supported and we're not 100% to Mocha ember wise yet.

File: style/ember/README.md [hunk 1]
```diff
@@ -2,3 +2,13 @@ Ember
 =====
 
 * Don't put a space between the opening handlebars braces and the variable.
+
+Testing
+-------
+
+* Visually separate the setup, exercise and assertion phases by:
```
Comment (BlakeWilliams) [inline style/ember/README.md]: Maybe remove this part since that's (I think) already a best practice and unnest the two rules below?

Comment (drapergeek) [inline style/ember/README.md]: :+1:

**Comment by jferris** [comment]
This has been open for a while without further comments. Ready to merge?

ping @seanpdoyle

**Comment by seanpdoyle** [comment]
@jferris I think so

**Comment by jferris** [comment]
@seanpdoyle merge away!

**Pull request merged**